### PR TITLE
Update troubleshooting.md and associated docs

### PR DESCRIPTION
**Pull request by rjmholt**
## PR Summary

Did some updating of our main documentation:
- Put myself into the maintainers list (😉)
- Updated `troubleshooting.md` and turned it into a full issue walkthrough document
- Changed the bug report template slightly to point to `troubleshooting.md`

Anything else I should add while I'm at it?

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -1,84 +1,294 @@
...
+
+If you believe there is a security vulnerability in the [PowerShell Extension]
+(or in [PowerShell Editor Services]), it **must** be reported directly to
+secure@microsoft.com to allow for [Coordinated Vulnerability Disclosure].
+**Only** open an issue if secure@microsoft.com has confirmed that filing
+an issue on GitHub is appropriate.
+
+[Editor Syntax]: https://github.com/PowerShell/EditorSyntax
+[PowerShell Editor Services]: https://github.com/PowerShell/PowerShellEditorServices
+[PowerShell Extension]: https:github.com/PowerShell/vscode-powershell/
```
Comment (SeeminglyScience) [inline docs/troubleshooting.md]: This link doesn't resolve, needs`https:githib.com` > `https://github.com`

Side note, I didn't know about this syntax! Really cool 😃

Comment (rjmholt) [reply docs/troubleshooting.md]: Aha! I wondered why it was breaking

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -20,28 +17,42 @@ The more repro details you can provide, along with a zip
...
+- Output from `$PSVersionTable`
 
 ```
 Copy / paste the following commands into the PowerShell Integrated Console, and paste the output here:
 
-code -v
-$pseditor.EditorServicesVersion
-code --list-extensions --show-versions
-$PSVersionTable
+> code -v
```
Comment (rkeithhill) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: By adding the `>` it makes it harder to copy/paste into the PSIC.  Now you have to manually remove `>` chars in order to execute the commands.  We might also want to mention that there is a PowerShell extension command `Upload Bug Report to Github` to generate a bug report that will automatically include all of the info above.  Well, when it works ... sigh.  That's another bug to look into.

Comment (rjmholt) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: Yeah I was vacillating between the two. I keep seeing issues where those commands aren't executed properly or the output is dumped into the issue with no formatting and is unreadable.

I was thinking maybe the `>` would prompt people to put the output underneath to break up the issue better... But maybe it's not a good idea?

There's a reference to the open bug command in troubleshooting.md, but I can put one in the template as well

Comment (rjmholt) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: Wait the `Upload Bug Report to GitHub` is already in the template -- second line in the comment

**Comment by rkeithhill** [comment]
Doh!  The pasteable script for getting env info works great if you have PSRL enabled.  But not so well if you don't.  That's one downside I guess to living/testing on the bleeding edge.  If we lose the starting `& {` and trailing `}` it will work pre-PSRL but it will have intermediate prompt strings and commands in the output.

**Comment by rjmholt** [comment]
I think I've got it fixed in `master` now -- took out the newlines and replaced them with semicolons.

**Comment by rjmholt** [comment]
(Tested in VSCode without PSRL first)

**Pull request merged**